tests/guestlog/guestlog.go

```diff
@@ -0,0 +1,249 @@
+package guestlog
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	expect "github.com/google/goexpect"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/pointer"
+	"kubevirt.io/kubevirt/tests"
+	"kubevirt.io/kubevirt/tests/clientcmd"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/libvmi"
+	"kubevirt.io/kubevirt/tests/testsuite"
+)
+
+const cirrosStartupTimeout = 60
+const testString = "GuestConsoleTest3413254123535234523"
+
+var _ = Describe("[sig-compute]Guest console log", decorators.SigCompute, Serial, func() {
```

Comment:
I don't see modifications to the KubeVirt CR anymore, so is `Serial` really still required?

Reply:
right, now we can finally avoid it